pkg/termination/write.go

```diff
@@ -55,3 +67,10 @@ func WriteMessage(path string, pro []v1alpha1.PipelineResourceResult) error {
 	}
 	return nil
 }
+
+//MessageLengthError indicate the length of termination message of container is beyond 4096 which is the max length read by kubenates
+type MessageLengthError string
+
+func (e MessageLengthError) Error() string {
+	return string(e)
```

Comment:
nit: The contents of this string will always be the same, so its message can also be a const. I'm pretty sure the compiler will notice this so it's not so much for performance (which is negligible anyway) but more for readability.

This would let line 53 become `return MessageLengthError{}` and hide the specifics of the string down here.

Reply:
Thanks @ImJasonH , I enhance the implements, please let me know if it satisfy you.